### PR TITLE
CDPT-2087 Remove old functionality

### PR DESCRIPTION
**Pull request by vertism**
## Description
Removal of:
- Watchlist
- Statistics
- Actionlist
- POD export
- Round robin
- Finance interest flag
- I will write
- PQ correction
- POD query
- Minister check fields
- Answer fields
- Quick action export controller

### CODE REVIEW

File: app/controllers/dashboard_controller.rb [hunk 1]
```diff
@@ -57,7 +57,6 @@ def load_pq_with_counts(dashboard_state)
     @statuses                 = @questions.all.map(&:state).uniq
     @question_types           = @questions.where("question_type != 'Follow-up IWW'").map(&:question_type).uniq
```
Comment (lucas-shaw) [inline app/controllers/dashboard_controller.rb:58]: Does this "Follow-up IWW" need to be deleted?

File: app/models/pq_scopes.rb
Comment (lucas-shaw) [inline app/models/pq_scopes.rb:1]: Same in the not_tx method, can "Follow-up IWW" be removed

File: app/assets/stylesheets/pq.scss
Comment (lucas-shaw) [inline app/assets/stylesheets/pq.scss:1]: Can this be removed on line 167?   

&.pod-query { background-color: #051EFF; }

**Comment by lucas-shaw** [comment]
Running `i18n-tasks unsued` shows a couple of translations that could be removed perhaps

```
activerecord.attributes.early_bird_organiser.date_from
activerecord.attributes.early_bird_organiser.date_to
```

File: app/controllers/dashboard_controller.rb (hunk 1 above)
Comment (vertism) [reply app/controllers/dashboard_controller.rb:58]: This value still exists in the database, so I have left this code in place.

**Review by lucas-shaw — APPROVED** [review]

**Pull request merged**